cube/Task.js

```diff
@@ -1,16 +1,80 @@
+const dateDiffInSeconds = (datetime) =>
+  `((DATE_PART('day', ${datetime}) * 24
+  + DATE_PART('hour', ${datetime})) * 60
+  + DATE_PART('minute', ${datetime})) * 60
+  + DATE_PART('second', ${datetime})`
+
+
+const dateDiffInMinutes = (datetime) =>
+  `(${dateDiffInSeconds(datetime)}) / 60`
+
+
 cube(`Task`, {
-  sql: `SELECT id, type, done_after AS after, done_before AS before, status FROM public.task`,
+  sql: `SELECT id, type, done_after, done_before, status FROM public.task`,
```

Comment:
Why did you remove the alias? Those columns should be renamed at some point (#535)

Reply:
I rename them when I create dimensions, so I thought it would be less confusion to have original names here

see https://github.com/coopcycle/coopcycle-web/pull/3107/files/abf8b28e867ea236875e15bc199d10f2486e07ec#diff-42bcc539c0025a538098703736d8be20df36864ca909db3ee39b3949c4fe4209R86